packages/client/src/Conversation.tsx

```diff
@@ -159,42 +170,80 @@ function Presentable({
   conversationId: string
   presentable: graphql.Presentable
 }) {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null)
   const [editing, setEditing] = React.useState(false)
+  const classes = useStyles()
+
+  function handleClick(event: React.MouseEvent<HTMLButtonElement>) {
+    setAnchorEl(event.currentTarget)
+  }
+
+  function handleClose() {
+    setAnchorEl(null)
+  }
+
+  //TODO: set aria-control's id
   return (
-    <div>
-      <strong>
-        <Participant {...presentable.from} />
-      </strong>{" "}
-      {moment(presentable.date).calendar()}{" "}
-      <PresentableEdited presentable={presentable} />
-      {presentable.contents.map((content, i) => {
-        if (editing) {
-          return (
-            <EditForm
-              accountId={accountId}
-              conversationId={conversationId}
-              contentToEdit={content}
-              onComplete={() => {
-                setEditing(false)
-              }}
-            />
-          )
-        } else {
-          return (
-            <>
-              <button
+    <Card className={classes.presentable}>
+      <CardHeader
+        title={displayParticipant(presentable.from)}
+        avatar={<Avatar address={presentable.from} />}
+        subheader={
+          moment(presentable.date).calendar() +
+          " " +
+          displayPresentableEdited(presentable)
```

Comment:
We don't want to display an edited message if the content has not been edited. Messages that have not been edited are showing "null" here.